ext/mysqli/mysqli.stub.php

```diff
@@ -56,9 +56,10 @@ public function commit(int $flags = -1, ?string $name = null) {}
     /**
      * @return mysqli|null|false
      * @alias mysqli_connect
+     * @no-verify
```

Comment:
```suggestion
     * @no-verify Should really be a static method
```
Or so, to make it more obvious why we're disabling verification?

Reply:
Yeah, good idaea!